teachopencadd/talktorials/002_compound_adme/talktorial.ipynb

```diff
@@ -6,11 +6,11 @@
    "source": [
```

Comment:
<ul><li><a href="http://www.swissadme.ch/" target="_blank">SwissADME&nbsp;</a> webserver</li><li>Description of 'lead compounds' ...&nbsp;</li><li>What is LogP?&nbsp;...</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/TeachOpenCADD/pull/49/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Agreed, that reads nicer!</p>